### ISSUE TITLE
PR builder: Cluster failed to initialize within 300 seconds

### ISSUE THREAD

**Issue by timstclair** [kubernetes/kubernetes#28641]
Happened several times in the last few runs:
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/28543/kubernetes-pull-build-test-e2e-gce/48158/
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/26696/kubernetes-pull-build-test-e2e-gce/48157/
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/27243/kubernetes-pull-build-test-e2e-gce/48156/

```
Waiting up to 300 seconds for cluster initialization.

  This will continually check to see if the API for kubernetes is reachable.
  This may time out if there was some uncaught error during start up.

...........................................Cluster failed to initialize within 300 seconds.
2016/07/07 15:09:32 e2e.go:218: Error running up: exit status 2
2016/07/07 15:09:32 e2e.go:214: Step 'up' finished in 7m56.450395031s
2016/07/07 15:09:32 e2e.go:114: Error starting e2e cluster. Aborting.
exit status 1
```

**Comment by timstclair** [comment]
@krousey @fejta

**Comment by timstclair** [comment]
Looks like a GCE issue:
```
ERROR: (gcloud.compute.firewall-rules.delete) Some requests did not succeed:
 - The resource 'projects/k8s-jkns-pr-gce/global/firewalls/e2e-gce-agent-pr-38-0-minion-e2e-gce-agent-pr-38-0-http-alt' was not found

ERROR: (gcloud.compute.firewall-rules.delete) Some requests did not succeed:
 - The resource 'projects/k8s-jkns-pr-gce/global/firewalls/e2e-gce-agent-pr-38-0-minion-e2e-gce-agent-pr-38-0-nodeports' was not found
```

A recent run failed with a slightly different error:
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/28639/kubernetes-pull-build-test-e2e-gce/48160/
```
ERROR: (gcloud.compute.instances.create) Some requests did not succeed:
 - The resource 'projects/google-containers/global/images/gci-dev-53-8530-6-0' is obsolete.  New uses are not allowed.  A suggested replacement is 'projects/google-containers/global/images/gci-dev-53-8490-0-0'.

Created [https://www.googleapis.com/compute/v1/projects/k8s-jkns-pr-gce/global/firewalls/e2e-gce-agent-pr-27-0-minion-all].
NAME                              NETWORK                SRC_RANGES     RULES                     SRC_TAGS  TARGET_TAGS
e2e-gce-agent-pr-27-0-minion-all  e2e-gce-agent-pr-27-0  10.180.0.0/14  tcp,udp,icmp,esp,ah,sctp            e2e-gce-agent-pr-27-0-minion
Some commands failed.
...
ERROR: (gcloud.compute.instances.describe) Could not fetch resource:
 - The resource 'projects/k8s-jkns-pr-gce/zones/us-central1-f/instances/e2e-gce-agent-pr-27-0-master' was not found

2016/07/07 15:14:40 e2e.go:218: Error running up: exit status 1
2016/07/07 15:14:40 e2e.go:214: Step 'up' finished in 2m0.195399639s
2016/07/07 15:14:40 e2e.go:114: Error starting e2e cluster. Aborting.
exit status 1
```

**Comment by timstclair** [comment]
Possibly a duplicate of https://github.com/kubernetes/kubernetes/issues/28612?

**Comment by timstclair** [comment]
This appears to have resolved itself.

**Comment by timstclair** [comment]
Happened again.

**Comment by ixdy** [comment]
Still flaking. cc @kubernetes/test-infra-maintainers

**Comment by gmarek** [comment]
@ixdy - can you paste the link to logs or a suite/run number?

**Comment by ixdy** [comment]
my own pet peeve! sorry about that.
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/29477/kubernetes-pull-build-test-e2e-gce/50321/ is the failing run.

**Comment by gmarek** [comment]
@dchen1107

**Comment by dchen1107** [comment]
The most recent failure reported by @ixdy has the error message in kubelet.log of the master node:
```
Failed to start cAdvisor inotify_add_watch /sys/fs/cgroup/memory/system.slice/kube-logrotate.service: no such file or directory caused kubelet not being able to sync pods.
```

I think it is a dup of https://github.com/kubernetes/kubernetes/issues/28997, which should be fixed by #29492

**Comment by ixdy** [comment]
Thanks @dchen1107! I should probably have also pointed out that the failure I linked was on a PR being cherry-picked into release-1.3, so it might be fixed in master already.
